recipes/xmidas/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "xmidas" %}
+{% set version = "0.1.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/xmidas-{{ version }}.tar.gz
+  sha256: d1650909e99ba18985d73ef6383639f777ce92f95e7092597d0c3889aad06692
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - xmidas = xmidas.main:start_xmidas
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - h5py
+    - numpy
+    - packaging
+    - pandas
+    - pyqtgraph >=0.12.2
+    - pyshortcuts
+    - pystackreg
+    - python >=3.7
+    - scikit-image
+    - scikit-learn
+    - scipy >=1.7
+    - silx ==0.15.2
+    - tifffile
+    - xraylarch >=0.9.57
+    - pyqt
+    - opencv
+
+test:
+  imports:
+    - xmidas
+    - xmidas.uis
+    - xmidas.main
+  commands:
+    - xmidas --help
+  requires:
+    - pip
```

Comment:
You're not running `pip check`, so this is uneeded. Or you should add `pip check` under commands above.